src/Paprika/Chain/Blockchain.cs

```diff
@@ -1055,6 +1070,22 @@ private void SetAccountRaw(in Keccak address, Span<byte> payload, bool newAccoun
             _touchedAccounts.Add(address);
         }
 
+        public void SetKeyForProof(in Key key, Span<byte> payLoad)
+        {
+            SetImpl(key, payLoad, EntryType.Proof, key.Type == DataType.Account ? _state : _storage);
+        }
+
+        public void RemoveMerkle(in Key key)
+        {
+            //Remove without marking hash dirty - dangerous!
```

Comment:
To a method summary? Maybe, rename the method to `UnsafeRemoveMerkle`? Also, `Debug.Assert` that the `Key` is actually `Merkle`?

Reply:
Removed the method - unused